system-variables.md

```diff
@@ -1492,22 +1587,24 @@ Usage example:
 SET tidb_query_log_max_len = 20
 ```
 
-### tidb_rc_read_check_ts (New in v6.0.0)
+### tidb_rc_read_check_ts <span class="version-mark">New in v6.0.0</span>
 
 > **Warning:**
 >
 > - This feature is incompatible with [`replica-read`](#tidb_replica_read-new-in-v40). Do not enable `tidb_rc_read_check_ts` and `replica-read` at the same time.
 > - If your client uses a cursor, it is not recommended to enable `tidb_rc_read_check_ts` in case that the previous batch of returned data has already been used by the client and the statement eventually fails.
 
 - Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
 - Default value: `OFF`
 - This variable is used to optimize the timestamp acquisition, which is suitable for scenarios with read-committed isolation level where read-write conflicts are rare. Enabling this variable can avoid the latency and cost of getting the global timestamp, and can optimize the transaction-level read latency.
 - If read-write conflicts are severe, enabling this feature will increase the cost and latency of getting the global timestamp, and might cause performance regression. For details, see [Read Committed isolation level](/transaction-isolation-levels.md#read-committed-isolation-level).
 
 ### tidb_read_staleness <span class="version-mark">New in v5.4.0</span>
 
 - Scope: SESSION
-- Default value: ""
+- Default value: `0`
+- Range: `[-2147483648, 0]`
```

Comment:
This is a bug fix in https://github.com/pingcap/tidb/issues/32763 - it is function neutral.